src/test/java/seedu/address/testutil/TypicalStudents.java

```diff
@@ -31,15 +31,15 @@
             .withAddress("123, Jurong West Ave 6, #08-111").withEmail("alice@example.com")
             .withPhone("94351253").withStudyLevel("Sec 2").withGuardianPhone("82813844")
             .withRelationship("Mother")
-                        .withSession(
+                        .addSession(
                                 new SessionBuilder().withSessionDate("2020-01-01", "12:00").build()
                         )
             .build();
     public static final Student BENSON = new StudentBuilder().withName("Benson Meier")
             .withAddress("311, Clementi Ave 2, #02-25")
             .withEmail("johnd@example.com").withPhone("98765432")
             .withStudyLevel("Primary 2").withGuardianPhone("81902144").withRelationship("Father")
-                        .withSession(
+                        .addSession(
```

Comment:
Only one comment, I was thinking the indentation for the addSession(withSession previously) was abit weird and should have the same lvl of indentation with withStudyLevel instead. Do you think it would be better if we did that?

Reply:
Oh yes, i also found the indentation weird as well. Will make the change, thanks!!